lib/fastlane/plugin/wpmreleasetoolkit/helper/an_metadata_update_helper.rb

```diff
@@ -125,6 +125,10 @@ def generate_block(fw)
       def extract_key(line)
         line.split(' ')[1].tr('\"', '')
       end
+
+      def is_int?(value)
+        true if Integer(string) rescue false
+      end
```

Comment:
This is not the superclass, but a sibling class.

`StandardMetadataBlock` is the superclass you want to move this method to.
Both `ReleaseNoteMetadataBlock` and `ReleaseNoteShortMetadataBlock` are subclasses of that so moving the `is_int?` from the latter to the former will make it work for the long release notes entry… but will break it for the short release notes; we need for this method to be declared on `StandardMetadataBlock` so that it's available on both subclasses instead.

Reply:
Hmm @AliSoftware can you clarify a bit, since we have 

kit/blob/8a1da4d702a53bc06a4f98cba9dac7ceee9111f0/lib/fastlane/plugin/wpmreleasetoolkit/helper/an_metadata_update_helper.rb#L134

Doesn't that make `ReleaseNoteMetadataBlock` the superclass of `ReleaseNoteShortMetadataBlock` and since the is_int function was only being used in those two classes I didn't see the need to push that out to `StandardMetadataBlock` or am I misunderstanding something in Ruby here?

Reply:
Ohhh right, I can't read 🤦  (should stop working so late ^^) Yep that should work then 👍